### DISCUSSION TITLE
Nutzung mit BFU möglich?

### DISCUSSION THREAD

**Discussion by max-102** [dewenni/ESP_Buderus_KM271#59] [Q&A] [answered]
Ich würde dieses Projekt gerne nutzen (mit dem Board von the78mole). Vorher wollte ich aber kurz fragen, ob eh auch weiterhin die Bedienung mit der BFU möglich ist? (Damit auch meine nicht so technikaffine Mutter weiterhin die Heizung steuern kann.) Über Erfahrungsberichte dazu würde ich mich freuen.

**Comment by JieffeEl** [comment]
[Sorry, I don't speak German]. I have both the board (for more than a year) and the BFU thermostat and they are both playing along nicely. So no problem having both. The status lights on the BFU are changing following MQTT requests to reflect the current state of the boiler.

**Reply by max-102** [reply]
Thanks for the reply on the status lights! How about the temperature? Can I set it with the board and if I turn the knob on the BFU later it is set to the value from the BFU?

**Reply by JieffeEl** [reply]
I'm not sure you can set the room temperature via MQTT or the web interface, only through the BFU knob. But Sven surely knows the answer to that question. To be honest, I never change the temperature, just switch between auto and manual modes.

**Comment by the78mole** [comment]
I'm 99% sure, that you can not set the current room temperature through other means than the BFU. We had this question quite often. I also don't know about a parameter that can be set for the actual value of the room temperature.

**Reply by xmonika** [reply]
I had the same idea - use the board to control Buderus by lowest temperature in the several zones instead of where the BFU is present.

I am checking the docs and guess I could manipulate the day setpoint temperature to force Buderus to turn the burner on and then turn it off when the temperature in the coldest zone will reach the target. Is this the right approach or are there better ways for this?

**Comment by HMMike** [comment, marked as answer]
Raumtemperatur läst sich bei aktivierter Fernbedienung nur an der BFU einstellen.
Ansonsten arbeitet das einwandfrei, ich habe die konstellation auch
